src/server/mod.rs

```diff
@@ -1,11 +1,8 @@
 //! The ipc-agent json rpc node.
 
-pub mod config;
 pub mod jsonrpc;
-mod request;
-mod response;
+pub mod request;
+pub mod response;
 
 const DEFAULT_JSON_RPC_SERVER_VERSION: &str = "2.0";
-pub const DEFAULT_JSON_RPC_SERVER_ENDPOINT: &str = "json_rpc";
-pub const DEFAULT_JSON_RPC_SERVER_PROTOCOL: &str = "http";
-pub const DEFAULT_JSON_RPC_SERVER_ADDR: &str = "127.0.0.1:3030";
+const DEFAULT_JSON_RPC_SERVER_ENDPOINT: &str = "json_rpc";
```

Comment:
As mentioned above, I would probably move these to the server config.

Reply:
I have moved them already